cnf-certification-test/preflight/suite.go

```diff
@@ -80,6 +80,8 @@ func testPreflightOperators(env *provider.TestEnvironment) {
 		}
 	}
 
+	logrus.Infof("Completed running preflight operator tests for %d operators", len(env.Operators))
```

Comment:
Could `env.Operators` be `nil`? If it is a nil slice, `len` returns `0`. Maybe log zero operators explicitly?

Reply:
It works to just check the length.  `len` handles the nil properly.

https://go.dev/play/p/xuipYDlnHx4

Reply:
`len` doesn't handle `nil` pointer:
 https://go.dev/play/p/5O0uDUlFiU4
 
 The code works. I thought about improving the logging:
```
if ops := len(env.Operators); ops > 0 {
  logrus.Infof("Completed running preflight operator tests for %d operators", ops)
else {
  logrus.Info("There is no operator to run preflight operator tests")
}
```

Reply:
No it doesn't.  The variable in your code is `untyped`.  This is why it fails.

Reply:
https://go.dev/play/p/Msjo6-ziBsY

Reply:
Yeah Go knows that `*int` isn't a slice so `len` doesn't handle it.